io_scene_niftools/modules/nif_import/armature/__init__.py

```diff
@@ -235,15 +235,20 @@ def guess_orientation(self, n_armature):
         # return string identifiers
         return ids[forward_ind], ids[up_ind]
 
+    @staticmethod
```

Comment:
What's the purpose of implementing this instead of using np.argmax? Avoiding the import?